### PR TITLE
added ns-o-ran-grafana tutorial

### PR DESCRIPTION
**Pull request by CogSP**
Addition of a new tutorial about [ns-o-ran-grafana](https://github.com/wineslab/ns-o-ran-grafana).

I added a new markdown file, following as much 
possible the style of the other tutorials and some images in assets/post-assets/

### CODE REVIEW

**Comment by Thecave3** [comment]
I will check it on my version and make a review there

**Comment by leonardobonati** [comment]
Thank you for adding this tutorial. Please double-check for typos.

File: _posts/tutorials/2024-02-25-ns-o-ran-grafana.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
+---
+layout: post
+title: "Graphical Visualization of simulated ns-3 data with Grafana"
+date: 2022-02-25
```
Comment (Thecave3) [inline _posts/tutorials/2024-02-25-ns-o-ran-grafana.md:4]:
```suggestion
date: 2024-02-25
```

File: _posts/tutorials/2024-02-25-ns-o-ran-grafana.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
+---
+layout: post
+title: "Graphical Visualization of simulated ns-3 data with Grafana"
```
Comment (Thecave3) [inline _posts/tutorials/2024-02-25-ns-o-ran-grafana.md:3]:
```suggestion
title: "Graphical Visualization of ns-O-RAN data with Grafana"
```

File: _posts/tutorials/2024-02-25-ns-o-ran-grafana.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+- User: admin 
+- Password: admin 
+and select a dashboard. On a terminal, run the python script
+
+```bash
+$ docker compose exec ns3 /bin/bash
+$ cd ns3-mmwave-oran
+$ python3 sim_watcher.py
+```
+This script is a watchdog that monitors the output files of the ns3 simulation within the directory. The data generated by ns3 is analyzed and sent via Telegraf to InfluxDB, from which Grafana extracts the data for visualization.
```
Comment (Thecave3) [inline _posts/tutorials/2024-02-25-ns-o-ran-grafana.md:48]:
```suggestion
This script is a watchdog that monitors the output files of the ns-3 simulation within the directory. The data generated by ns-3 is collected and sent to InfluxDB through Telegraf, which is then pulled by Grafana for visualization. 
```

File: _posts/tutorials/2024-02-25-ns-o-ran-grafana.md [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+
+
+### Dashboards
+In Grafana, go to 'Dashboard' then 'Manage'. 
+
+
+<img src="/assets/post-assets/grafana_manage_dashboard_option.png" class="post-image" alt="Grafana Manage Dashboard Option" width="30%" >
+
+
+Four dashboard are available to be displayed:
```
Comment (Thecave3) [inline _posts/tutorials/2024-02-25-ns-o-ran-grafana.md:83]:
```suggestion
Four dashboards are available:
```

File: _posts/tutorials/2024-02-25-ns-o-ran-grafana.md [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+```
+or the entire tables with queries. For instance:
+```bash
+> select * from /L3servingSINR3gpp_cell_[0-9]/
+```
+show the tables containing SINR3 values measured at the network (L3) level for the 5G cells.
+
+
+### Dashboards
+In Grafana, go to 'Dashboard' then 'Manage'.
```
Comment (Thecave3) [inline _posts/tutorials/2024-02-25-ns-o-ran-grafana.md:77]: Rewrite the introduction

**Review by Thecave3 — CHANGES_REQUESTED** [review]
The tutorial is fine, but I would check a little bit better the language.

File: _posts/tutorials/2024-02-25-ns-o-ran-grafana.md (hunk 5 above)
Comment (CogSP) [reply _posts/tutorials/2024-02-25-ns-o-ran-grafana.md:77]: Do you mean this part?

"This tutorial demonstrates the process of visualizing ns-3 simulation data through Grafana. The following example involve extracting data from an ns-3 simulation and transmitting it to an InfluxDB server using Telegraf, in order for Grafana to collect and present the data on a dashboard, facilitating convenient analysis and interpretation. Four distinct dashboards have been designed to showcase the data derived from “cu-up,” “cu-cp,” and “du” files within the simulation container. The fourth dashboard specifically plots aggregated data points of interest. All the code is available in [this repository](https://github.com/wineslab/ns-o-ran-grafana)."

Comment (CogSP) [reply _posts/tutorials/2024-02-25-ns-o-ran-grafana.md:77]: I think this is better:

"This tutorial illustrates how to visualize ns-O-RAN simulation data using Grafana. The presented example extracts information from an ns-3 simulation, sending it to InfluxDB through Telegraf. This process enables Grafana to pull and display the data on a dashboard, streamlining analysis and interpretation. Four dashboards were created, with three specifically designed to visualize data extracted from the simulation's "cu-up," "cu-cp," and "du" files. The fourth dashboard is dedicated to plotting aggregated points of interest. All the code is available in [this repository](https://github.com/wineslab/ns-o-ran-grafana)."

Comment (Thecave3) [reply _posts/tutorials/2024-02-25-ns-o-ran-grafana.md:77]: Better, thanks implement this

**Review by Thecave3 — APPROVED** [review]

**Pull request merged**